crates/meta/src/types.rs

```diff
@@ -202,18 +193,27 @@ impl BqlType {
             decimal_item if decimal_item.starts_with(b"Decimal") => {
                 Self::_decimal_type(decimal_item)
             },
+            fixed_string_item if fixed_string_item.starts_with(b"FixedString") => {
+                match &fixed_string_item[b"FixedString".len()..] {
+                    [b'(', len @ .., b')'] => {
+                        Ok(BqlType::FixedString(Self::_parse_num(len)?))
+                    }
+                    _ => Err(MetaError::UnknownBqlTypeConversionError),
+                }
+            }
             _ => Err(MetaError::UnknownBqlTypeConversionError),
         }
     }
 
+    fn _parse_num(bytes: &[u8]) -> MetaResult<u8> {
+        str::from_utf8(bytes)
```

Comment:
methods in string/str are expensive although this may not cause the perf downgrade in obvious looking in this case. I suggest you use itoa to do a simple and much fast parsing from bytes. 

Search "itoa::write" in TB to give more example.